.github/workflows/yehezgun.yml

```
@@ -0,0 +1,38 @@
+name: Template Git Actions
+
+run-name:
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  Base-Actions:
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "🎉 The job was automatically triggered by a ${{ github.event_name }} event"
+
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: "yarn"
+
+      - name: Install dependencies
+        run: yarn --immutable
```

Are we using Yarn 2 in this project? I thought we're using Yarn Classic.

```suggestion
        run: yarn --frozen-lockfile
```

---

Still yarn version`1.22.19`

---

Then should we use `yarn --frozen-lockfile` instead of `--immutable`?

---

Updated